### PR TITLE
feat: updates context menu shortcut for code nav

### PR DESCRIPTION
**Pull request by andrialexandrou**
### Why:

Closes: 

No tracking issue, that I'm aware of

### What's being changed (if available, include any code snippets, screenshots, or gifs):

We needed to change the keyboard shortcut in Code Nav. Relevant conversation is in https://github.com/github/accessibility-audits/issues/4102. Now the docs need to be updated. 

### Check off the following:

- [x] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline.

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by cmwilson21** [comment]
👋 @andrialexandrou - Thanks so much for submitting a PR! I'll get this triaged for review :zap:

File: content/get-started/using-github/keyboard-shortcuts.md [hunk 1]
```diff
@@ -94,7 +94,7 @@ If you view a code file in a repository using the **Code** view and click on any
 | Keyboard shortcut | Description
 |-----------|------------
 |<kbd>Shift</kbd>+<kbd>J</kbd>| Highlights the line that is currently selected by the cursor within the code file
-|<kbd>Shift</kbd>+<kbd>F10</kbd> | If a line of code is currently selected, this shortcut opens the line menu for that line, appearing under {% octicon "kebab-horizontal" aria-label="The horizontal kebab icon" %} to the left of the line
+|<kbd>Option</kbd> (Mac)/<kbd>Alt</kbd> (Windows) + <kbd>Shift</kbd>+<kbd>C</kbd> | If a line of code is currently selected, this shortcut opens the line menu for that line, appearing under {% octicon "kebab-horizontal" aria-label="The horizontal kebab icon" %} to the left of the line
```
Comment (hubwriter) [inline content/get-started/using-github/keyboard-shortcuts.md:97]: This isn't expressed in the way we normally do things. Better to keep the whole key combo together:

```suggestion
|<kbd>Shift</kbd>+<kbd>Option</kbd>+<kbd>C</kbd> (Mac) or </br> <kbd>Shift</kbd>+<kbd>Alt</kbd>+<kbd>C</kbd> (Windows/Linux) | If a line of code is currently selected, this shortcut opens the line menu for that line, appearing under {% octicon "kebab-horizontal" aria-label="The horizontal kebab icon" %} to the left of the line
```

**Review by hubwriter — CHANGES_REQUESTED** [review]
@andrialexandrou - Thanks for raising this PR. 🎉 

One little suggested change. If this look ok please commit the suggestion and we'll get this merged.

Thanks again 👍

**Comment by andrialexandrou** [comment]
Reviewed: https://docs-27307-a742a2.preview.ghdocs.com/en/get-started/using-github/keyboard-shortcuts#navigating-within-code-files

**Review by hubwriter — APPROVED** [review]

**Pull request merged**